manifests/helper.pp

```diff
@@ -0,0 +1,53 @@
+# @summary manage a conntrack helper
+#
+# @example FTP helper
+#  nftables::helper { 'ftp-standard':
+#    content => 'type "ftp" protocol tcp;',
+#  }
+#
+# @param content
+#   Conntrack helper definition.
+# @param table
+#   The name of the table to add this helper to.
+# @param helper
+#   The symbolic name for the helper.
+define nftables::helper (
+  # lint:ignore:parameter_documentation
```

Comment:
Why is this needed? Looks like you have documented them.

Reply:
I copied it from an existing define. let me drop it

Reply:
Did you forget to push your changes?

Reply:
it seems that way. sigh. Hopefully that time is a charm